ark/utils/data_utils.py

```diff
@@ -22,9 +22,10 @@ def generate_deepcell_input(data_xr, data_dir, nuc_channels, mem_channels):
         mem_channels (list):
             membrane channels to be summed over
     Raises:
-            ValueError:
-                Raised if nuc_channels and mem_channels are both None or empty
+        ValueError:
```

Comment:
So does this work with one of them set to None, or it fails?

Reply:
It works if one of them is set to `None`. I realize there was an obscure issue with my testing that was causing this to fail, not Omer's logic.